### PR TITLE
Adding alternate solution for HTTP Collect

### PR DESCRIPTION
**Pull request by SufiyanSamnani**
This solution doesn't require the BufferList module for the HTTP Collect challenge. The existing solution only shows an example solution with the BufferList module and a solution without using the module isn't included in LearnYouNode yet.

### CODE REVIEW

File: exercises/http_collect/solution/solution_alt.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const http = require('http')
+let outputData = ''
+
+http.get(process.argv[2], function (response) {
```
Comment (ccarruitero) [inline exercises/http_collect/solution/solution_alt.js]: I think if we use es6 syntax for variables in the solution, should use es6 arrow functions too

**Comment by ccarruitero** [comment]
Hi @SufiyanSamnani , your solution looks good. Just a minor comment.

Aside an alternative solution should be inside [test folder](https://github.com/workshopper/learnyounode/tree/master/test/http_collect)

**Comment by shanmukhateja** [comment]
@ccarruitero what needs to be done to get this merged?

**Comment by ccarruitero** [comment]
@shanmukhateja mainly, the solution should be moved to [test folder](https://github.com/workshopper/learnyounode/tree/master/test/http_collect). This way can be tested with ci.

**Pull request closed without merging**